### PR TITLE
GROW-141 Monthly good promo cards in homepage loan category area

### PR DESCRIPTION
**Pull request by ryan-ludwig**

### CODE REVIEW

**Comment by eddieferrer** [comment]
Looks good, except for the styling. 

@ryan-ludwig Note, that the requirements on GROW-141 mention that COVID should to go `https://www.kiva.org/covid19response`

**Comment by ryan-ludwig** [comment]
@eddieferrer Ended up doing some refactoring on PromoGridLoanCard.vue to allow custom routes and prevent double background image request on retina.

File: src/components/Homepage/LendByCategory/LoanCategoriesSection.vue [hunk 1]
```diff
@@ -33,6 +33,7 @@
 		>
 			<loan-category
 				:loans="getCategoryLoans(category.id)"
+				:category-name="cleanCategoryName(category)"
```
Comment (eddieferrer) [inline src/components/Homepage/LendByCategory/LoanCategoriesSection.vue:36]: We don't need to add this additional prop. See my comment below

File: src/components/Homepage/LendByCategory/LoanCategory.vue [hunk 2]
```diff
@@ -140,6 +156,16 @@ export default {
 			// remove any text contained within square brackets, including the brackets
 			return String(this.name).replace(/\s\[.*\]/g, '');
 		},
+		monthlyGoodPromoData() {
```
Comment (eddieferrer) [inline src/components/Homepage/LendByCategory/LoanCategory.vue:159]: Without the categoryName prop, you can do this same switch logic using:
`this.name` or `this.id` ( I added the id in my commit for GROW-189

File: src/components/Homepage/LendByCategory/FeaturedLoansCarousel.vue [hunk 3]
```diff
@@ -298,7 +299,7 @@ h3 {
 
 	// Overwrite styles for loan card to make it responsive.
 	::v-deep .lend-homepage-loan-card {
-		width: 93%;
+		margin: 1rem;
```
Comment (ryan-ludwig) [inline src/components/Homepage/LendByCategory/FeaturedLoansCarousel.vue:302]: This keeps the box-shadow always visible at smaller sizes

File: src/pages/MonthlyGood/MonthlyGoodLandingPage.vue [hunk 4]
```diff
@@ -103,6 +103,9 @@ const pageQuery = gql`query monthlyGoodLandingPage {
 const heroImagesRequire = require.context('@/assets/images/mg-landing-hero', true);
 
 export default {
+	metaInfo: {
```
Comment (ryan-ludwig) [inline src/pages/MonthlyGood/MonthlyGoodLandingPage.vue:106]: Sneaking this in, noticed while testing the monthly good promo cards

**Pull request merged**